pandas-stubs/core/indexes/interval.pyi

```diff
@@ -122,6 +217,30 @@ class IntervalIndex(IntervalMixin, ExtensionIndex):
     def get_value(self, series: ABCSeries, key): ...
     @property
     def is_all_dates(self) -> bool: ...
+    @overload  # type: ignore[override]
+    def __gt__(self, other: Interval | IntervalIndex) -> np_ndarray_bool: ...  # type: ignore[misc]
+    @overload
+    def __gt__(self, other: object) -> bool: ...
```

Comment:
Don't think we need to allow `IntervalIndex` to be compared to any other object for `__gt__`, __ge__`, etc

Reply:
Fixed.

Reply:
Didn't see this as fixed?

Reply:
Pretty sure this is gone from the latest.